src/components/v5/shared/UserPopover/partials/UserInfo.tsx

```diff
@@ -65,15 +65,14 @@ const UserInfo: FC<UserInfoProps> = ({
               text={formatText({ id: 'userInfo.top.contributor.in' })}
             />
             <div className="flex gap-1">
-              {domains
-                ?.slice(0, 3)
-                .map(({ domainName, domainId }) => (
+              {domains?.slice(0, 3).map(({ domainName, domainId }) => (
+                <div key={domainId}>
```

Comment:
any particular reason we wrap this in a div now?

Reply:
I had a ts error, but now it's gone, so I'm reverting the old changes.